ete-test/tcframe/GenerationEteTests.cpp

```diff
@@ -25,6 +25,17 @@ TEST_F(GenerationEteTests, Normal) {
     ));
 }
 
+TEST_F(GenerationEteTests, NotGeneratingOutput) {
```

Comment:
`Normal_NoOutput`.

Change the slug to `normal-no-output`.

Also need `multi-no-output` as well. See my other comment on how you missed this for problems with multiple test cases in a file.